src/core/library.h

```diff
@@ -236,6 +236,12 @@ typedef struct QUIC_LIBRARY {
     const uint32_t* DefaultCompatibilityList;
     uint32_t DefaultCompatibilityListLength;
 
+    //
+    // Last sample of the performance counters
+    //
+    uint64_t PerfCounterSamplesTime;
+    int64_t PerfCounterSamples[QUIC_PERF_COUNTER_MAX];
```

Comment:
Probably worth wrapping this, and everything else in the telemetry assert define check.

Reply:
Never mind. Eventually I may want to DoS protection stuff here too.